sympy/parsing/latex/latex_grammar.py

```diff
@@ -0,0 +1,2059 @@
+# The file was automatically generated by Lark v0.8.1
+#
+#
+#   Lark Stand-alone Generator Tool
+# ----------------------------------
+# Generates a stand-alone LALR(1) parser with a standard lexer
+#
+# Git:    https://github.com/erezsh/lark
+# Author: Erez Shinan (erezshin@gmail.com)
+#
+#
+#    >>> LICENSE
+#
+#    This tool and its generated code use a separate license from Lark.
+#
+#    It is licensed under GPLv2 or above.
```

Comment:
Standalone parsers generated with 0.8.2+ are licensed under the MPL, which seems more compatible with BSD-3...

https://github.com/lark-parser/lark/issues/507

Also there have been some performance enhancements...

Reply:
Great I will make sure to use the latest! Also nice to hear that the license is more compatible

Reply:
I didn't notice this. GPL is definitely a no go. I'm not sure if MPL is OK either actually. I don't understand why they would restrict the licensing of the parser like this.